published/012-2016-August-24.md

```diff
@@ -0,0 +1,130 @@
+# IPFS Newsletter #11
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened since the IPFS team week in Lisbon in late July. This is the first Newsletter in a while, and there's a ton of exciting content below.
+
+For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [newsletter](//tinyletter.com/ipfsweekly)!
+
+## Updates
+
```

Comment:
A table of contents might help the reader skim through and go directly to what they find interesting